include/ck/tensor_operation/gpu/device/device_put_element.hpp

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
```

Comment:
Add some comments to indicate this file is not used

Reply:
I add the put example in 0bf57502f260e12ff9ca2cff78424c36c8a0a507